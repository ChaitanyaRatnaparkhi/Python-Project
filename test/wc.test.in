Hello, my name is Chaitanya this is a normal sentence.