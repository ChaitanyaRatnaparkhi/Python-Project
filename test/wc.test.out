        1       10      54      test/wc.test.in
